subsystem/subsystem-core/pom.xml

```diff
@@ -111,6 +111,11 @@
             <version>1.2.101.v20150831-1342</version>
         </dependency>
         <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
Was discussed with John Ross in https://issues.apache.org/jira/browse/ARIES-1565